.github/workflows/email.yml

```diff
@@ -64,16 +64,16 @@ jobs:
               Branch: ${{github.ref}} <br>
               Revision: ${{ github.event.pull_request.merge_commit_sha }}  <br>
               Author: ${{ env.AUTHOR}} <br>
-              Link: ${{env.LINK}} <br>             
-              Log Message: <br>
+              Link: ${{env.LOG}} &nbsp;&nbsp;${{github.event.pull_request._links.html.href}} <br>             
```

Comment:
To address "I’d like the link to point to the PR merged and not to a commit"
https://github.com/chapel-lang/chapel/actions/runs/3323581443/jobs/5494106859 "pull_request": {
    "_links": {
      "comments": {
        "href": "https://api.github.com/repos/chapel-lang/chapel/issues/20913/comments"
      },
      "commits": {
        "href": "https://api.github.com/repos/chapel-lang/chapel/pulls/20913/commits"
      },
      "html": {
        "href": "https://github.com/chapel-lang/chapel/pull/20913"
      },